docs: update the readme to latest working one (#908)
bnb-chain/node: README.md

- Issuing new digital assets (we have a standard called BEP-2)
- Mint/burn, freeze/unfreeze, lock/unlock of digital assets

> Besides it get many other rich features:

Reviewer: `get` to `gets` or `has` ?

Author: fixed, change to `has`

* Full nodes are not validator nodes, but anyone can get a copy of the whole blockchain and validate it.
* No PoW means block times are very fast.
* UXTO/account does not matter as we just use the [cosmos](https://github.com/cosmos/cosmos-sdk/tree/master/x/bank) bank.
* Features like the DEX (deprecated now) will run directly on the node as apps written in golang.
[Read](https://tendermint.readthedocs.io/en/master/introduction.html) [more](https://blog.cosmos.network/tendermint-explained-bringing-bft-based-pos-to-the-public-blockchain-domain-f22e274a0fdb) about Tendermint and ABCI.

## Getting Started

### Environment setup

#### Requirement
Go version above 1.17 is required.

> Please [install it](https://golang.org/dl) or use brew on macOS: `brew install go` and `brew install dep`.

Reviewer 2: `dep` should be unnecessary now

Author: fixed

> Please [install it](https://golang.org/dl) or use brew on macOS: `brew install go` and `brew install dep`.

Reviewer 2: ```suggestion
Please [install it](https://go.dev/doc/install) or use brew on macOS: `brew install go`.
```

* Full nodes are not validator nodes, but anyone can get a copy of the whole blockchain and validate it.
* No PoW means block times are very fast.
* UXTO/account does not matter as we just use the [cosmos](https://github.com/cosmos/cosmos-sdk/tree/master/x/bank) bank.

> * Features like the DEX will run directly on the node as apps written in golang(deprecated now).

Reviewer: `golang(deprecated now)` -> `golang (deprecated now)`

Author: fixed

Assets may be issued through `bnbcli` while the blockchain is running; see here for an example:

```bash

> $ chainId=`cat ~/.bnbchaind/config/genesis.json  |jq .chain_id --raw-output`

Reviewer 2: ```suggestion
$ chainId=`cat ~/.bnbchaind/config/genesis.json | jq .chain_id --raw-output`
```

Start your node, then list your keys as below:

```bash
$ ./build/bnbcli keys list 

NAME:   TYPE:   ADDRESS:                                                PUBKEY:
testnode        local   bnb1elavnu4uyzt43hw380vhw0m2zl7djz0xeac60t      bnbp1addwnpepqva5fmn4r4hc66fpqafwwdf20nq8xjpr3kezkclpmluufxdk5x4gw9xsln5
```

Check a balance with this command, e.g.:

```bash

> $  ./build/bnbcli account bnb1elavnu4uyzt43hw380vhw0m2zl7djz0xeac60t  --chain-id ${chainId} |jq

Reviewer 2: ```suggestion
$  ./build/bnbcli account bnb1elavnu4uyzt43hw380vhw0m2zl7djz0xeac60t --chain-id ${chainId} | jq
```

```

You can look at the contents of the tx, use the tx hash above:

```bash
$  ./build/bnbcli tx 40C2B27B056A63DFE5BCE32709F160F53633C0EBEBBD05E1AC26419D35303765 --chain-id ${chainId} 
```

Then you can check the balance of pepe's key to see that he now has 100 satoshi units of `FBTC-4C9`:

```bash

> $ ./build/bnbcli account bnb1u5mvgkqt9rmj4fut60rnpqfv0a865pwnn90v9q  --chain-id ${chainId} |jq

Reviewer 2: ```suggestion
$ ./build/bnbcli account bnb1u5mvgkqt9rmj4fut60rnpqfv0a865pwnn90v9q  --chain-id ${chainId} | jq
```
